colcon_metadata/metadata/__init__.py

```diff
@@ -46,3 +49,67 @@ def get_metadata_files(path=None):
             path = os.path.join(dirpath, filename)
             files.append(path)
     return files
+
+
+def merge_metadata(dst_dict, other_dict, meta_source):
```

Comment:
`other_dict` -> `src_dict` might make it more clear.

Reply:
Renamed: 4aeba30f2d58aee2f9f92c82b25858ac0b4db235.